### ISSUE TITLE
Ghost button outline fails WCAG 2.1 non-text contrast requirements

### ISSUE THREAD

**Issue by corinedwards** [themesberg/flow-ui#1]
The outline install button is failing WCAG 2.1 non-text contrast requirements :(

![Screen Shot 2020-02-27 at 5 10 20 pm](https://user-images.githubusercontent.com/1154708/75420622-21739700-5984-11ea-9350-573ed09c8ef8.png)

**Comment by zoltanszogyenyi** [comment]
Thanks @corinedwards We'll have this fixed while in development.
